exampleCourse/questions/demo/annotated/engCircuit/question.html

```diff
@@ -2,22 +2,22 @@
 
 <p>Determine the {{params.ask}} of the circuit below:</p>
 
-<p> <pl-figure file-name={{params.figname}} directory="clientFilesQuestion"></pl-figure> </p>
+<p><pl-figure file-name="figure.svg" type="dynamic" width="750px"></pl-figure></p>
```

Comment:
It's a bit nitpicking, but maybe change the width to something a bit more relative? e.g., 100%?

Reply:
Actually, one more thing: it may be good to add an `alt` for accessibility reasons.

Reply:
Changed to 100% and added dynamic alt text here https://github.com/PrairieLearn/PrairieLearn/pull/8179/commits/85e0c6b9be23c804969d39dee155703ecbaaa412